design/KruizeLocalAPI.md

```diff
@@ -2746,8 +2746,9 @@ If no experiment type is specified, it will default to `container`.
 **Request with `model_settings` and `term_settings` field**
 
 Under `recommendation_settings`, the `model_settings` and `term_settings` fields are optional 
-but can be used to specify model and term details. Currently, for monitoring mode user can set
-model as cost and/or performance and terms can be set to short, medium and/or long term.
+but can be used to specify model and term details. Currently, for monitoring mode user can either set
+model as cost and/or performance or can specify a custom model by defining cpu, memory and gpu percentiles.
+Terms can be set to short, medium and/or long term.
 By default, it provides recommendations for all three terms and both models.
```

Comment:
Can you please rewrite this entire paragraph and focus on the "why" first and then the "how". For a first time user, it is not clear why is this needed. Also please reference the example values in the yaml below to help explain the "how".